app/main.py

```diff
@@ -34,45 +34,45 @@ class Bridge:
     def __init__(self):
         """Initialise a Bridge object."""
         self._rabbit_hosts = [
-            settings.RABBIT_HOST,
-            settings.RABBIT_HOST2
+            settings.SDX_GATEWAY_EQ_RABBITMQ_HOST,
+            settings.SDX_GATEWAY_EQ_RABBITMQ_HOST2
         ]
-        self._rabbit_port = settings.RABBIT_PORT
-        self._default_user = settings.DEFAULT_USER
-        self._default_pass = settings.DEFAULT_PASSWORD
+        self._eq_queue_rabbit_port = settings.SDX_GATEWAY_EQ_RABBIT_PORT
+        self._eq_queue_user = settings.SDX_GATEWAY_EQ_RABBITMQ_USER
+        self._eq_queue_password = settings.SDX_GATEWAY_EQ_RABBITMQ_PASSWORD
 
-        self._corp_rabbit_port = settings.AWS_CORP_RABBITMQ_PORT
-        self._aws_user = settings.AWS_CORP_USER
-        self._aws_pass = settings.AWS_CORP_PASSWORD
+        self._sdx_queue_rabbit_port = settings.SDX_GATEWAY_SDX_RABBITMQ_PORT
```

Comment:
I think we should drop `rabbit` from the property names to be consistent with the user and port properties.